runtime/JavaScript/spec/rewriter/Makefile

```diff
@@ -0,0 +1,15 @@
+ANTLR_JAR = .antlr/antlr-4.12.0-complete.jar
+
+.antlr:
+	mkdir .antlr
+
+.antlr/antlr-4.12.0-complete.jar: .antlr
```

Comment:
please avoid hard coded versions, use variables instead to ease the release process

Reply:
Good point, fixed.